WebSearch/websearch.ini

```diff
@@ -177,6 +177,11 @@ history_keep = all
 [predefined_site/GitHub]
 url = https://github.com/search?q=%s
 
+[predefined_site/MDN]
+url = https://developer.mozilla.org/search?q=%s
+home_url = https://developer.mozilla.org/

```

Comment:
`home_url` is unnecessary in this case since it is equal to the root of `url`

Reply:
`home_url` line removed!